katacoda/get-started/03-accessing/Dockerfile.template

```diff
@@ -1,6 +1,5 @@
-FROM dvcorg/doc-katacoda:base
+FROM ${TAG_PREFIX}/doc-katacoda:base
 
-ARG RELEASE_HASH 
```

Comment:
why remove? (also same with all other similar cases...)

Reply:
Because these are set by environment variables now, with `envsubst` in `generate.bash`. Previously `RELEASE_HASH` was supplied as `--build-arg` in `docker build`. `ARG` in `Dockerfile` is misleading.